### ISSUE TITLE
Submission Package bug - PT_95S99RWE not in diagnoses manifest

### ISSUE THREAD

**Issue by chris-s-friedman** [d3b-center/d3b-cds-manifest-prep#120]
### Describe the bug

PT_95S99RWE is not in the diagnosis manifest

### Expected behavior

PT_95S99RWE should have a diagnosis

### Version ID

0.13.0

### Effected file(s)

- [ ] sample
- [ ] participant
- [X] diagnosis
- [X] diagnosis_sample_mapping
- [ ] file
- [ ] file_sample_participant_map
- [ ] genomic_info

**Comment by chris-s-friedman** [comment]
tldr: there are no diagnoses for the event related to the samples for this participant

There are two samples being released in CDS for this participant, `BS_KSFM87N1` and `BS_YR7Q70YP`.  Both of these samples are normal samples. They are from sample `7316-65`. There are no other aliquots from this sample. Because of this, there are no tumor samples associated with this sample, so, in the histologies file, there is no diagnosis associated with this event. So, there is no diagnosis to associate here. 

Unless we use diagnoses from a different event... which breaks our model for where diagnoses come from.

**Comment by chris-s-friedman** [comment]
there are other diagnoses from the other sample relate to this participant. Excerpt from the histologies file: 

| Kids_First_Biospecimen_ID | Kids_First_Participant_ID | sample_id | pathology_diagnosis | broad_histology | pathology_free_text_diagnosis | sample_type |
|---------------------------|---------------------------|-----------|---------------------|-----------------|-------------------------------|-------------|
| BS_9D7YPY72               | PT_95S99RWE               | 7316-323  | Craniopharyngioma   |                 | craniopharyngioma             | Tumor       |
| BS_CV8J6WK8               | PT_95S99RWE               | 7316-323  | Craniopharyngioma   |                 | craniopharyngioma             | Tumor       |
| BS_HYGK88B0               | PT_95S99RWE               | 7316-323  | Craniopharyngioma   |                 | craniopharyngioma             | Tumor       |
| BS_KSFM87N1               | PT_95S99RWE               | 7316-65   |                     |                 |                               | Normal      |
| BS_YR7Q70YP               | PT_95S99RWE               | 7316-65   |                     |                 |                               | Normal      |
| BS_YS76FT7P               | PT_95S99RWE               | 7316-323  | Craniopharyngioma   |                 | craniopharyngioma             | Tumor       |
